src/booking-list/components/CalendarItemsWrapper.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { bookingItemsManager } from 'src/booking-list/BookingItemsManager';
+import { CalendarItems } from 'src/booking-list/components/CalendarItems';
+import { datesManager } from 'src/booking-list/datesManager';
+import { useBookingRides } from 'src/booking-list/useBookingRides';
+
+export const CalendarItemsWrapper = () => {
+  const requestRides = useBookingRides();
```

Comment:
variable declarations should go without line breaks if these declarations are short. If a declaration is multiple lines, then we do a newline before and after such a long declaration. This is just a convention and not a rule

Reply:
did